bluesky_database/frontend/src/app/__tests__/csv-export.test.tsx

```
@@ -0,0 +1,513 @@
+
+
+// Mock data structure matching the page component
+interface Post {
+  id: string
+  timestamp: string
+  username: string
+  text: string
+}
+
+// Extract and test the CSV export function directly
+const exportToCSV = (posts: Post[]) => {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'exportToCSV'. Consider to delete it or rename it.

'exportToCSV' is defined here:


(lint/suspicious/noRedeclare)